### PR TITLE
Adds Rock Token to In App Browser

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Screen Shot 2019-10-11 at 10 55 35 AM

### What does this PR do, or why is it needed?

Adds the ability to include the Rock user token as a query parameter to the URL of the in-app browser.

### What design trade-offs/decisions were made?

Left the positional arguments to not introduce breaking changes but they should really be keyword arguments.

### How do I test this PR?

Go to the test panel and click "Open InAppBroswer with Rock Token"

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes [tag-issues-here]
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
Does this feature work in all rock instances 😲

**Comment by redreceipt** [comment]
> Does this feature work in all rock instances 😲

I think so...

**Comment by redreceipt** [comment]
can't test at the moment because apollosrock is down :)

File: packages/apolloschurchapp/src/ui/WebBrowser/Browser.js [hunk 1]
```diff
@@ -3,39 +3,50 @@ import InAppBrowser from 'react-native-inappbrowser-reborn';
...
 };
 
 const Browser = {
-  open: async (url, options) => {
-    const cookie = await getCookie();
-    if (cookie)
+  open: async (
+    baseURL,
+    options,
+    auth = { useRockCookie: false, useRockToken: false }
```
Comment (vinnyjth) [inline packages/apolloschurchapp/src/ui/WebBrowser/Browser.js]: smart making this `false` by default,  I would hate to leak this to third party sites.

**Pull request merged**